app/routes/notifications.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
```

Comment:
Please make use of `AuthenticatedRouteMixin` to ensure access to logged in users only.And please add title via titleToken()